kibbeh/src/stories/UserBadge.story.tsx

```diff
@@ -9,14 +9,14 @@ export default {
 
 const TheUserBadge: Story = () => {
   return (
-    <div className="flex flex flex-row">
-      <div className="flex m-1">
+    <div className="flex flex-row">
+      <div className="flex m-0">
         <UserBadge>ƉC</UserBadge>
       </div>
-      <div className="flex m-1">
+      <div className="flex m-0">
         <UserBadge>ƉS</UserBadge>
       </div>
-      <div className="flex m-1">
+      <div className="flex m-0">
```

Comment:
why
```suggestion
      <div className="flex">
```

Reply:
Because the previous UserBadge implementation had way too much spacing and didn't fit the exact model of the figma design.